src/main/java/org/testng/xml/Parser.java

```diff
@@ -70,14 +65,14 @@ public Parser(String fileName) {
    * found in the classpath.
    */
   public Parser() throws FileNotFoundException {
-    init(null, null, null);
+    init(null, null);
   }
 
   public Parser(InputStream is) {
-    init(null, is, null);
```

Comment:
These changes come from another pull-request. Could you rebase your branch with master?

Reply:
Sorry, I will handle it.